Automattic/jetpack
# AI Assistant: Add Breve under feature flag

Fixes #38018

## Proposed changes:

* Adds Breve as-is under a feature flag, in the Jetpack AI sidebar

Currently in JavaScript. This PR is intended to add Breve with just enough changes for it to work as the POC on its current `main` branch.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Enable Breve (https://github.com/Automattic/jetpack/pull/37986)
* Check that the current functionality is added in the sidebar
* Disable the feature flag
* Check that the feature is disabled
